apps/voting/contracts/Voting.sol

```diff
@@ -432,28 +438,28 @@ contract Voting is IForwarder, AragonApp {
     }
 
     /**
-     * @notice Return the sliced list of delegated voters for `_delegate` and their voting power at the current block
+     * @notice Return the sliced list of delegated voters for `_delegate`
      * @param _delegate the address of the delegate
      * @param _offset the number of delegated voters from the start of the list to skip
      * @param _limit the number of delegated voters to return
      * @return the array of delegated voters
-     * @return the array of voting power of delegated voters
      */
-    function getDelegatedVoters(address _delegate, uint256 _offset, uint256 _limit) external view returns (address[] memory, uint256[] memory) {
-        return _getDelegatedVotersAt(_delegate, _offset, _limit, getBlockNumber64());
-    }
+    function getDelegatedVoters(address _delegate, uint256 _offset, uint256 _limit) external view returns (address[] memory voters) {
+        require(_delegate != address(0), ERROR_ZERO_ADDRESS_PASSED);
 
-    /**
-     * @notice Return the sliced list of delegated voters for `_delegate` and their voting power at the `_voteId` snapshot block
-     * @param _delegate the address of the delegate
-     * @param _offset the number of delegated voters from the start of the list to skip
-     * @param _limit the number of delegated voters to return
-     * @return the array of delegated voters
-     * @return the array of voting power of delegated voters
-     */
-    function getDelegatedVotersAtVote(address _delegate, uint256 _offset, uint256 _limit, uint256 _voteId) external view voteExists(_voteId) returns (address[] memory, uint256[] memory) {
-        Vote storage vote_ = votes[_voteId];
-        return _getDelegatedVotersAt(_delegate, _offset, _limit, vote_.snapshotBlock);
+        address[] storage votersList = delegatedVoters[_delegate].addresses;
+        uint256 votersCount = votersList.length;
+        if (_offset >= votersCount) {
+            return voters;
+        }
+
+        uint256 returnCount = _offset.add(_limit) > votersCount ? votersCount.sub(_offset) : _limit;
+        voters = new address[](returnCount);
+        for (uint256 i = 0; i < returnCount; ++i) {
+            address voter = votersList[_offset + i];
+            voters[i] = voter;
+        }
+        return voters;
```

Comment:
No need to return `voters` explicitly.

Reply:
Addressed in f1f6de1606a6eda1999a0e1cc0ea895857101f88.